persistence/schema/gov.go

```diff
@@ -0,0 +1,124 @@
+package schema
+
+const ( // TODO (Team) - I'm sure there's obvious optimizations here
```

Comment:
Long message incoming...

So this is weird because:
1. Params is a map, and not a structure SQL schema
2. There is only ever 1 record: the current set of params
3. This essentially create a SQL table that will only ever have 1 row

Though we are getting out of scope, the answer as to how we want to store this is the exact same as to how we want to store feature flags.

Per [this StackOverflow answer](https://stackoverflow.com/a/43264444/768439), we can go with either either `hstor`, `json` or `jsonb`. However, after reading [this](https://www.citusdata.com/blog/2016/07/14/choosing-nosql-hstore-json-jsonb/), the tl;dr is that we should use `jsonb` for key-value maps stored within json.

I understand why you'd want to explicitly list out all the columns here, but ultimately the source of truth for the "schema" of the government params is the protobuf, so my suggestion would be to:
1. Create a table called `params` or `maps`
2. Params will have a column `featureFlags JSONB NOT NULL` and `govParame JSON NOT NULL`

This loses the schema validation benefit at the database engine layer, but I can argue (preferably not over text) as to why that's fine.

Reply:
By `So this is weird because`, I implied `I'm not sure what the best approach here so here are just some thoughts I have`.

Per a suggestion from @iajrz offline, this schema was suggested

```
param values table
|------------+------------|
| field name | field type |
|------------+------------|
| height     | bigint     |
| param_id   | int        |
| value      | string     |
|------------+------------|


params table
|------------+------------|
| field name | field type |
|------------+------------|
| id         | int        |
| value_type | int        |
| param_name | string     |
|------------+------------|
```

My modification to it would be:

```

CREATE TYPE val_type AS ENUM ('STRING', 'BIGINT', 'SMALLINT');

CREATE TABLE params (
  name    string     PRIMARY_KEY,
  height  bigint     PRIMARY_KEY,
  enabled bool       NOT NULL,
  type    val_type   NOT NULL,
  value   string     NOT NULL,
)

CREATE TABLE flags (
  name    string     PRIMARY_KEY,
  height  bigint     PRIMARY_KEY,
  enabled bool       NOT NULL,
  type    val_type   NOT NULL,
  value   string     NOT NULL,
)
```

We can have very similar tables for flags and params and not how the primary key is a composite key of the name of the flag/param and the height, along with an enabled/disabled bool, so we can automatically (for example) in one snapshot of the state, predefined the behaviour of a flag only being applied for some number of blocks.

Reply:
#76